scripts/main.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+import os
+import json
+from jinja2 import Environment, FileSystemLoader
+
+def convert_region_json():
+    with open('./src/GeositeFramework/region.json') as f:
```

Comment:
All the file path references in these scripts can be made portable between mac/linux/windows by using the [os.path](https://docs.python.org/2/library/os.path.html) module.  The `os.path.join` calls will add some noise, but cross platform compatibility seems valuable, especially if it's easily had with a few modifications.

In doing so, we can also add some of the parts to constants and not have the path strings sprinkled around.  Something along the lines of this:

```python
BASE_DIR = os.path.join('src', 'GeositeFramework')
REGION_FILE = os.path.join(BASE_DIR, 'region.json')
TMPL_FILE = os.path.join(BASE_DIR, 'template_index.html)
...
```

If they're used across scripts, consider defining them in a file in the module so they could be imported everywhere.

Reply:
Good suggestion I didn't think about! Building for all platforms is tricky. I anticipate the file refs will only be used here -- but should that change, porting them to a constants file is trivial.